models/DataObject/ClassDefinition/Data/ReverseObjectRelation.php

```diff
@@ -199,7 +199,7 @@ public function getFieldType(): string
 
     public function getClasses(): array
     {
-        if($this->ownerClassId) {
+        if($this->getOwnerClassId()) {
```

Comment:
After testing and reviewing, I believe this is the best solution to keep the previous logic while safe calling ownerClassId so that it is not null.